src/main/app/src/static/smart-documents-result.html

```diff
@@ -0,0 +1,357 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2024 Lifely
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, viewport-fit=cover"
+    />
+
+    <title>SmartDocuments wizard resultaatpagina</title>
+
+    <!-- Preload critical assets for performance -->
+    <link
+      rel="preload"
+      href="/assets/fonts/Roboto/400.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preload"
+      href="/assets/fonts/Roboto/500.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preload"
+      href="/assets/i18n/nl.json"
+      as="fetch"
+      crossorigin="anonymous"
+    />
+
+    <!-- Optimized CSS -->
+    <style>
```

Comment:
Does it make sense to extract this to css file (and preload it if needed)?
Is this a good practice?

Reply:
Might be useful to have CSS and JS in separate files if we reuse them. Since this is not the case I'm ok having the code as it is now.